### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by lilskyex0x**
## Checklist for this Pull Request
🙌Kindly give a code review for this PR👨‍💻

In this milestone, I made changes and updates as listed below:

- [ ] Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- [ ] Created a directory for reusable components: `/components`.
- [ ] Installed [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- [ ] [Setup router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of `<App>`
- [ ] Added 2 routes in the application, each rendering a page component:
   -  `/`
   -  `/categories`
- [ ] Created components inside of the `/components` directory:
   - Contains individual book states, as well as a button to delete the book
   - Contains a list to render individual books
   - Contains a form to create new books
   - Contains navigation to link to the created 2 routes

💙Thank you

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (Ghiftee) [inline src/App.js]: - [ ] Kindly ensure you do not disable linters as they help us write cleaner code.

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (Ghiftee) [inline src/components/Book.js]: - [ ] Kindly ensure you do not disable linters as they help us write cleaner code.

File: src/components/BookList.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-named-as-default */
```
Comment (Ghiftee) [inline src/components/BookList.js]: - [ ] Kindly ensure you do not disable linters as they help us write cleaner code.

File: src/components/Categories.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-alert */
```
Comment (Ghiftee) [inline src/components/Categories.js]: - [ ] Kindly ensure you do not disable linters as they help us write cleaner code.

File: src/components/Links.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (Ghiftee) [inline src/components/Links.js]: - [ ] Kindly ensure you do not disable linters as they help us write cleaner code.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @lilskyex0x 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ Project is initiated using CRA
✔️ Routes are created
✔️ Book component is created

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 1 above)
Comment (Munsa1) [inline src/App.js]: - [ ] As requested by the previous reviewer, kindly remove the `disable-linters` as the rules help us to write industry-standard code

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @lilskyex0x , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @lilskyex0x, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Correct use of React-router ✔️ 
- Correct use of React components ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**